gui/default/syncthing/device/editDeviceModalView.html

```diff
@@ -40,14 +37,6 @@
                 <span translate ng-if="deviceEditor.deviceID.$error.unique && deviceEditor.deviceID.$dirty">A device with that ID is already added.</span>
               </p>
             </div>
-            <div ng-if="editingDeviceExisting()" class="input-group">
-              <div class="well well-sm text-monospace form-control" style="height: auto;" select-on-click>{{currentDevice.deviceID}}</div>
```

Comment:
I see no need to change this, the existing devices can still use the "Well" element, while adding uses the regular input (which gets set to read-only when adding from the notification).

Reply:
Is there any purpose not to use a read-only `input` element though? As far as I see it, it makes no difference regarding functionality and design (and semantically `input` is the appropriate element anyway).

Reply:
I remember now, it had to do with wrapping mainly.  The well word-wraps the ID nicely on narrow screens, while the input just truncates it.  That's an important distinction when looking at an existing device and trying to find out its ID.  But again, when adding a suggested ID, the user only wants it to be the one from the notification.  The need to see every character of it vanishes.

Having said that, why not just use the original code for read-only (the .well thing) also in case it's added from a notification?  Seems appropriate because you need none of the other stuff, namely the discovery list.

Reply:
The .well thing has been reused now, but the code has also been moved to the main section instead of having a separate one at the bottom. I'm marking this as "resolved" as the same conversation is already going on above, so I think it's better to keep everything in one place.